_layouts/standalone_resource.html

```diff
@@ -0,0 +1,345 @@
+
+   <div class="minimal-header-container default-grid">
+      {% include minimal-header.html translations = translations 
+                                     title = page.standalone_resource_header.title
+                                     subtitle = page.standalone_resource_header.subtitle 
+                                     link = page.standalone_resource_header.link %}
+    </div>
+
+    <div class="default-grid nav-container nav-page-specific">
+        <div class="nav">
+          {% if page.standalone_resource_pager %}
+          {% assign pager = page.standalone_resource_pager %}
+          <nav class="standalone-resource-pager" aria-label="Item navigation">
+            <ul>
+                <li>
+                {%- if page.standalone_resource_pager.icon -%}
+                    {%- capture icon-element -%}<img src="{{page.standalone_resource_pager.icon}}" alt="" class="category-icon">{%- endcapture -%}
+                {%- endif -%}
+                {%- if page.permalink == pager.ref %}
+                    <span class="pagename">{{icon-element}}{{pager.name}}</span>
+                {%else%}
+                    <a href="{{pager.ref}}">{{icon-element}}{{pager.name}}</a>
+                {%endif%}
+                </li>
+
+                {% if page.standalone_resource_pager.include %}
+                    {% include {{ page.standalone_resource_pager.include }} %}
+                    {% if context %}
+                        <li><a href="{{ context[1] }}">{% include_cached icon.html class="pager-icon" name="arrow-up-thin" %} {{ context[0] }}: {{context[2]}}</a></li>
+                    {% endif %}
+                    {% if prev  %}
+                        <li><a href="{{ prev[1] }}">{% include_cached icon.html class="pager-icon" name="arrow-left-thin" %} {{ prev[0] }}</a></li>
+                    {% endif %}
+                    {% if next %}
+                        <li><a href="{{ next[1] }}">{{ next[0] }} {% include_cached icon.html class="pager-icon" name="arrow-right-thin" %}</a></li>
+                    {% endif %}
+                {% endif %}
+            </ul>
+          </nav>
+        {% elsif page.standalone_resource_nav_links %}
+           <nav class="nav" aria-label="Context for this document">
+            <ul>
+            {% for link in page.standalone_resource_nav_links %}
+                <li class="nav__item"><a href="{{ link.ref | relative_url }}"{% if page.permalink == link.ref %} class="active" aria-current="page"{% endif %}>{{ link.name }}</a></li>
+            {% endfor %}
+            <li class="nav__item"><a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">
+                All WCAG 2 Guidance
+                {% include_cached icon.html name="different-view" %}
+            </a></li>
+            </ul>
+          </nav>
+          {% endif %}
+        </div>
+    </div>
+    <div class="default-grid with-gap leftcol">
+
+<style>
```

Comment:
Not good practice to put this into inline stylesheets. There's now an act-styles.css file that should be loaded into every page. Better to put it in there.

Reply:
Why not good practice specifically? Does confuse the cascade for sure. Separate file means separate network load by browser. 

These styles are NOT specific to ACT rule but common to anything that uses the standalone_resource layout. This was how Hidde had it ad I've not reworked to move them to the theme CSS .

Reply:
> This was how Hidde had it and I've not reworked to move them to the theme CSS.

@WilcoFiers , This seems this is a non-essential change request. I hope you're OK putting it as an issue to consider later. Or, now that you understand it applies more broadly, you are OK leaving it as is.